modules/admin/src/resources/scss/components/_folders.scss

```diff
@@ -41,6 +40,16 @@ $icon-line-height: $icon-font-size + 5px;
     .folders-items {
         padding-left: 3px;
     }
+
+    .is-movable {
+        .folders-label {
+            background-color: map-get($grays, 300);
+        }
+
+        .folders-actions-right {
+            display: block !important;
```

Comment:
@TheMaaarc maybe i am wrong, but whenever i see !important definition, it seems to be not well structured/defined css/html. What do you think?

Reply:
@nadar In this case you might be right. It's hard to say.
@dev7ch Can you try to add the `display: block;` on line 169 to the `.folders-action-right class`?

```
.folders-actions-right {
    .is-movable &,
    &.is-visible,
    &:hover {
        display: block;
    }
}
```

Maybe consider to add a direct child selector `>` to prevent the `display: block` from being applied to all childs (I know that it's not important for this specific case but good practice anyway).

And while you are on it, maybe you can restructure the whole file to work like that (see line 152 or 96 for example).